### DISCUSSION TITLE
Input Widgets in HTML

### DISCUSSION THREAD

**Discussion by tfabacher-dp** [daohoangson/flutter_widget_from_html#1204] [Ideas]
We are looking to use the flutter_widget_from_htm to create block HTML widgets to represent something like Monday.com's Automation snippets. It is a very easy way for users to input complex concepts and workflow in your applications. We would need to put in "input tags" and use flutter widgets for inputs [text, number and currency amounts] and dropdown select list.

Has anyone added input tags and used custom Flutter widget yet???

Here is a video of the idea:
https://support.monday.com/hc/en-us/articles/360001222900-monday-com-Automations

**Comment by daohoangson** [comment]
It should be possible, you will need a custom widget though. There was some discussion about this at https://github.com/daohoangson/flutter_widget_from_html/issues/1
